nucypher/crypto/keystore.py

```diff
@@ -342,15 +342,27 @@ def restore(cls, words: str, password: str, keystore_dir: Optional[Path] = None)
         return keystore
 
     @classmethod
-    def generate(cls, password: str, keystore_dir: Optional[Path] = None, interactive: bool = True) -> 'Keystore':
+    def generate(
+            cls, password: str,
+            keystore_dir: Optional[Path] = None,
+            interactive: bool = True,
+            report_mnemonic: bool = False
+            ) -> 'Keystore':
         """Generate a new nucypher keystore for use with characters"""
+        if report_mnemonic and interactive:
+            raise ValueError("The two values: report_mnemonic and interactive,  may not both be `True`")
+
         mnemonic = Mnemonic(_MNEMONIC_LANGUAGE)
         __words = mnemonic.generate(strength=_ENTROPY_BITS)
         if interactive:
             cls._confirm_generate(__words)
         __secret = bytes(mnemonic.to_entropy(__words))
         path = Keystore.__save(secret=__secret, password=password, keystore_dir=keystore_dir)
         keystore = cls(keystore_path=path)
+
+        if report_mnemonic:
```

Comment:
Aside from being literal, I think the name `return_mnemonic` will help avoid confusion with the existing reporting (console echo) behavior.

Reply:
I've changed `report_mnemonic` to  `return_mnemonic`.